### PR TITLE
Add tests to Coingecko provider

### PR DESCRIPTION
**Pull request by lexich**
Add tests to Coingecko provider

### CODE REVIEW

File: src/providers/conversion/Coingecko.ts [hunk 1]
```diff
@@ -20,14 +18,19 @@ export class Coingecko implements ConversionRateProvider {
...
   async getRate(from: Currency | Currency[], to: Currency | Currency[]) {
     const fromArray = typeof from === 'string' ? [from] : from;
     const toArray = typeof to === 'string' ? [to] : to;
     const fromIds = fromArray.map((currency) => Coingecko.translateCurrency(currency)).join(',');
     const toIds = toArray.map((currency) => Coingecko.translateCurrency(currency)).join(',');
-    const url = `https://api.coingecko.com/api/v3/simple/price?ids=${fromIds}&vs_currencies=${toIds}`;
-    const response = await axios(url);
-    const data = await response.data;
+    const data = await this.load(fromIds, toIds);
     return fromArray.reduce((previousPairs, fromCurrency) => {
```
Comment (m-sebastiyan) [inline src/providers/conversion/Coingecko.ts:34]: To remove typecasting:
```suggestion
    return fromArray.reduce<ConversionRatePair[]>((previousPairs, fromCurrency) => {
```

File: src/providers/conversion/Coingecko.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Currency } from './ConversionRateProvider';
+import { Coingecko } from './Coingecko';
+describe('Codegecko', () => {
+  let instance: Coingecko;
+  beforeEach(() => {
+    instance = new Coingecko();
+  });
+
+  it('getRate', async () => {
+    instance.load = () => {
```
Comment (m-sebastiyan) [inline src/providers/conversion/Coingecko.test.ts:10]: What about using a spy? 

https://github.com/metaplex/js/blob/main/test/actions/mintNFT.test.ts#L18

File: src/providers/conversion/Coingecko.ts (hunk 1 above)
Comment (lexich) [reply src/providers/conversion/Coingecko.ts]: Sure

File: src/providers/conversion/Coingecko.test.ts (hunk 2 above)
Comment (lexich) [reply src/providers/conversion/Coingecko.test.ts]: If you prefer this way, I'll update test

**Review by kurpav — APPROVED** [review]

**Pull request merged**